packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -389,38 +423,31 @@ class TxForm extends Component {
     }
 
     try {
-      const {
-        account: { address, transactionCount },
-        chainId,
-        ethBalance,
-        token
-      } = this.props;
-
-      if (!chainId) {
-        throw new Error('chaindId is required for an EthereumTx');
-      }
+      const { token } = this.props;
 
-      if (!address) {
-        throw new Error('address of an account is required');
-      }
+      const preValidation = this.preValidate(values);
 
-      if (!transactionCount) {
-        throw new Error('transactionCount is required for an EthereumTx');
+      // preValidate return an error if a field isn't valid
+      if (preValidation !== true) {
+        return preValidation;
       }
 
-      if (!token || !token.address || !token.decimals) {
-        throw new Error('token information is required for an EthereumTx');
+      // The 3 values below (`chainId`, `ethBalance`, and `transactionCount`)
+      // come from props, and are passed into `values` via the form's
+      // initialValues. As such, they don't have visible fields, so putting an
+      // error here just means we're keeping the form state as not valid.
+      if (!values.chainId) {
+        return { chainId: 'Fetching chainId' };
```

Comment:
Previously I found that the validation errors only appear in the UI if when the key is `amount` (i.e. `{ amount: 'some message' }`, I have not figured out why, so if you comment out the if statement, and run the code, it will not display 'Fetching chainId'.

Since it may be shown to the user who don't need to know about variable names, suggest changing it to `Fetching chain ID`, i.e. `return { amount: 'Fetching chain ID' };`.